nibabel/gifti/parse_gifti_fast.py

```diff
@@ -58,27 +56,29 @@ def read_data_block(encoding, endian, ordering, datatype, shape, data):
         newarr = np.fromstring(zdec, dtype=dt)
         if len(newarr.shape) != len(sh):
             newarr = newarr.reshape(sh, order=ord)
+
     elif enclabel == 'External':
         # GIFTI_ENCODING_EXTBIN
         raise NotImplementedError("In what format are the external files?")
+
     else:
         return 0
+
     # check if we need to byteswap
     required_byteorder = gifti_endian_codes.byteorder[endian]
     if (required_byteorder in ('big', 'little') and
-        required_byteorder != sys.byteorder):
+            required_byteorder != sys.byteorder):
```

Comment:
I think this would cause a(nother) PEP8 issue? Is this the preferred resolution of the competing styles?